packages/js/src/utils/errors.ts

```diff
@@ -1,19 +1,42 @@
 import { NormalizeKeyPaths } from '@silverhand/essentials';
+import { get } from 'lodash';
 
 const logtoErrorCodes = Object.freeze({
-  idToken: {
-    invalidIat: 'Invalid issued at time',
-    invalidToken: 'Invalid token',
+  id_token: {
+    invalid_iat: 'Invalid issued at time',
+    invalid_token: 'Invalid token',
+  },
+  callback_uri_verification: {
+    redirect_uri_mismatched: 'Redirect URI mismatched',
+    error_found: 'Error found',
+    missing_state: 'Missing state',
+    state_mismatched: 'State mismatched',
+    missing_code: 'Missing code',
   },
 });
 
 export type LogtoErrorCode = NormalizeKeyPaths<typeof logtoErrorCodes>;
 
+const getMessageByErrorCode = (errorCode: LogtoErrorCode): string => {
+  // TODO: linear issue LOG-1419
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  const message = get(logtoErrorCodes, errorCode, '');
```

Comment:
if we manually fallback to `errorCode` then the third param is not needed, otherwise `return errorCode` will never be executed.

Reply:
Removed 3rd parameter, PTAL.